{
    "category": "BAD_PRACTICE",
    "vtype": "DE_MIGHT_IGNORE",
    "priority": "3",
    "rank": "19",
    "project": "pool",
    "warning_line": "} catch(Exception e) {\n",
    "warning_method": "public synchronized void clear() {\nif (null != _factory) {\nIterator it = _pool.iterator();\nwhile (it.hasNext()) {\ntry {\n_factory.destroyObject(it.next());\n} catch (Exception e) {\n}\n}\n}\n_pool.clear();\n}"
}